web/docs/understand/architecture/components.md

```diff
@@ -138,3 +138,43 @@ using a pluggable *writer* for a given output format, e.g., JSON, CSV, or PCAP.
 This component is not yet implemented. Until then, the [SINK](#SINK)
 performs both output formatting and subsequent I/O.
 :::
+
+## Component Interaction
+
+This section provides examples of how the different compentents interact.
+
+### Ingestion
+
+Sending data to VAST (aka *ingesting*) involves spinning up a VAST client
+that parses and ships the data to a [VAST server](/docs/use/run):
+
+![Ingest process](/img/ingest.light.png#gh-light-mode-only)
+![Ingest process](/img/ingest.dark.png#gh-dark-mode-only)
+
+VAST first acquires data through a *carrier* that represents the data transport
+medium. This typically involves I/O and has the effect of slicing the data into
+chunks of bytes. Thereafter, the *format* determines how to parse the bytes into
+structured events. On the VAST server, a partition builder (1) creates
+sketches for accelerating querying, and (2) creates a *store* instance by
+transforming the in-memory Arrow representation into an on-disk format, e.g.,
+Parquet.
```

Comment:
This is missing the dense indexes. You could probably put this into (1) by just saying "query-accelerating index structure" or similar, because the reader probably doesn't know what a sketch is, but they are likely to have a good picture of what an index is.

Instead of partition builder it probably makes sense to just say something along the lines of "sorts by schema and creates respective (1) and (2)", and then also to explain the cut-off mechanism in number of events or partition timeout.

Reply:
Rephrased. I also added a link to the tuning section where the cut-off mechanics are explained.